staging/src/k8s.io/component-base/metrics/summary.go

```diff
@@ -99,12 +99,19 @@ type SummaryVec struct {
 func NewSummaryVec(opts *SummaryOpts, labels []string) *SummaryVec {
 	opts.StabilityLevel.setDefaults()
 
+	fqName := BuildFQName(opts.Namespace, opts.Subsystem, opts.Name)
```

Comment:
maybe we should have a method on opts which is opts.FQName()? (not in this PR though).